src/com/base/game/dialogue/places/dominion/lilayashome/Library.java

```diff
@@ -28,16 +28,100 @@ public String getContent() {
 		}
 
 		@Override
-		public Response getResponse(int index) {
-			if (index == 1) {
+		public Response getResponse(int main) {
+			if (main == 1) {
+				return new Response("Browse the Stacks", "Read one of the many books availble in the library.", BROWSE_BOOKS) {
+				};
+
+			} else if (main == 2) {
+				return new Response("Interact with Slaves", "Not yet implimented", null) {
+				};
+
+			} else {
+				return null;
+			}
+		}
+	};
+	
+
+	public static final DialogueNodeOld BROWSE_BOOKS = new DialogueNodeOld("", "", false) {
+		/**
+		 */
+		private static final long serialVersionUID = 1L;
+
+		@Override
+		public String getLabel() {
+			return "Library";
+		}
+
+		@Override
+		public String getContent() {
+			return "<p>"
+					+ "Walking down one of the aisles, you see a great deal of organization has gone into the design of the room. It clear that someone has spent a great deal of cleaning the room."
+					+ " As you walk you scan the titles printed onto the spines of the books, but there's not really much that catches your eye."
+					+ " Only a few shelves really look to be worth your time, and you wonder if you should take some time to do a spot of reading..."
+					+ "</p>";
+							
+		}
+
+		@Override
+		public Response getResponse(int books) {
+			if (books == 1) {
+				return new Response("General Knowledge", "A section of the library dedicated to books on common subjects", LORE_BOOKS);
+
+			}  else if (books == 2) {
+				return new Response("Races of Dominion", "A section of the library dedicated to books on predominate races within the city", DOMINION_RACES) {
+				};
+
+			} else if (books == 3) {
+				return new Response("The Fields", "A section of the library dedicated to books on the area known as the Foloi Fields", FIELDS_BOOKS) {
+				};
+//
+//			} else if (books == 4) {
+//				return new Response("The Seas", "A section of the library dedicated to books on the area known as the Endless Sea", SEA_BOOKS) {
+//				};
+//
+//			} else if (books == 5) {
+//				return new Response("The Jungle", "A section of the library dedicated to books on the area known as the Jungle", JUNGLE_BOOKS) {
+//				};
+//
+//			} else if (books == 6) {
+//				return new Response("The Desert", "A section of the library dedicated to books on the area known as the Desert", DESERT_BOOKS) {
+//				};
+
+			} else {
+				return null;
+			}
+		}
+	};
+	
+	public static final DialogueNodeOld LORE_BOOKS = new DialogueNodeOld("", "", false) {
+		/**
+		 */
+		private static final long serialVersionUID = 1L;
+
+		@Override
+		public String getLabel() {
+			return "Library";
+		}
+
+		@Override
+		public String getContent() {
+			return "<p>"
+					+ "Walking down one of the aisles, you see that these shelves have an arcane look to them. The shelves glow and shift before your eyes."
+					+ "The enhanced sight you seem to have gained allows you to see layers of arcane essance has been built up on these shelves. Some of the texts almost look like"
```

Comment:
"The enhanced sight you seem to have gained allows you to see layers of arcane essance has been built up"

something about this seems off... but I'm not sure. Is there a "that" missing somewhere or is it just the many different tenses making this look weird